testing/test-utils/src/lib/utils/dynamic-mocks/eslint-audits.mock.ts

```diff
@@ -1,5 +1,17 @@
 import type { AuditReport } from '@code-pushup/models';
 
+export const ESLINT_AUDITS_FIXED_SLUGS: (keyof typeof ESLINT_AUDITS_MAP)[] = [
```

Comment:
Can we reduce the total number of audits used in these integration tests? I am worried about the maintenance. Test data should be minimal while meeting diversity requirements - do e.g. all 9 of these audits display different behaviour?

Reply:
I would make this part of the issue to reduce number of ESLint audits - #558